### PR TITLE
PP-7794 split remaining app pipelines #1

### PR DESCRIPTION
**Pull request by mrlumbu**
Splits frontend, connector, ledger, adminusers and products pipelines as per pattern established by toolbox. Results have been applied so you can look at concourse to see if things look right

### CODE REVIEW

**Comment by oswaldquek** [comment]
Is [cardid](https://cd.gds-reliability.engineering/teams/pay-dev/pipelines/apps-test-ecr?group=cardid) excluded from this?

File: ci/pipelines/apps-test-ecr.yml [hunk 1]
```diff
@@ -25,6 +25,13 @@ resources:
       branch: master
       paths:
         - ci/pipelines/apps-test-ecr.yml
+  - name: pay-ci
```
Comment (oswaldquek) [inline ci/pipelines/apps-test-ecr.yml:28]: Might be for another PR but we could probably get rid of the `apps-test-ecr` resource now there is a `pay-ci` one.

**Review by oswaldquek — COMMENTED** [review]
One comment which doesn't stop the approval of this PR, but looks good. 
Not all the builds pass on concourse at the moment - I'm not sure if they need to at this stage so am holding off approving until I get clarity on that first.

File: ci/pipelines/apps-test-ecr.yml (hunk 1 above)
Comment (mrlumbu) [reply ci/pipelines/apps-test-ecr.yml:28]: yep, I'll tidy this up in another PR

**Comment by mrlumbu** [comment]
The builds weren't passing because staging ecr repos didn't exist. They do now, and I have kicked off all the failed pipelines so should go green soon

**Review by oswaldquek — APPROVED** [review]
Builds are green now 🎫

**Pull request merged**